sport/app/rugby/model/rugby.scala

```diff
@@ -52,29 +54,48 @@ object Stage extends Enumeration(1) {
   type Stage = Value
   val Group, KnockOut = Value
 }
-
 object Match {
   val dateFormat: DateTimeFormatter = DateTimeFormat.forPattern("yyyy/MM/dd").withZoneUTC()
+  implicit val writes: OWrites[Match] = (m: Match) =>
+    Json.obj(
+      "date" -> dateFormat.print(m.date),
+      "id" -> m.id,
+      "homeTeam" -> m.homeTeam,
+      "awayTeam" -> m.awayTeam,
+      "venue" -> m.venue,
+      "competitionName" -> m.competitionName,
+      "status" -> m.status,
+      "event" -> m.event,
+      "stage" -> m.stage,
+    )
```

Comment:
Lots of boilerplate / repetition here… it’s basically the match object with the date formatted?

Any benefit of using `m` over `match` as the variable name?